skbio/sequence/_sequence.py

```diff
@@ -547,6 +547,122 @@ def observed_chars(self):
     def _string(self):
         return self._bytes.tostring()
 
+    @classmethod
+    @experimental(as_of="0.4.0-dev")
+    def concat(cls, seqs, how='strict'):
+        """ Concatenate an iterable of Sequence objects.
```

Comment:
Can you describe how `metadata` is handled somewhere in this docstring? Maybe in a Notes section.